### PR TITLE
feat: 状态管理代码编辑UI优化

### PR DESCRIPTION
**Pull request by gene9831**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Background and solution

**before**
image
**after**
image

**before**
image
**after**
image

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by kevinmoch — DISMISSED** [review]
LGTM

**Review by wenmine — DISMISSED** [review]
LGTM

File: packages/plugins/data/src/CreateVariable.vue [hunk 1]
```diff
@@ -224,6 +217,17 @@ export default {
       }
     )
 
+    watch([() => props.createData.name, () => props.createData.variable], () => {
```
Comment (chilingling) [inline packages/plugins/data/src/CreateVariable.vue]: 直接 return 数组就 OK 了
watch(
  () => [a, b],
  () => {}
)

File: packages/plugins/data/src/CreateVariable.vue [hunk 2]
```diff
@@ -516,11 +473,19 @@ export default {
...
     &,
     &::after {
       border-bottom-color: var(--ti-lowcode-data-example-bg-color);
     }
   }
+
+  &[x-placement^='top'] .popper__arrow {
+    &,
+    &::after {
+      border-top-color: var(--ti-lowcode-data-example-bg-color);
```
Comment (chilingling) [inline packages/plugins/data/src/CreateVariable.vue:492]: 这个是不是全局的 popover 的箭头颜色都有问题，如果是的话，建议在全局修复

Comment (gene9831) [reply packages/plugins/data/src/CreateVariable.vue:492]: popover弹出框包括一个框和一个小箭头，这个箭头可能向上也可能向下。这里是自定义popover弹出框的背景色，重写了框的背景色和箭头背景色，并不是箭头颜色有问题

**Review by chilingling — APPROVED** [review]

**Pull request merged**